src/lib/LastManStandingUsage.ts

```diff
@@ -0,0 +1,246 @@
+	'{1} grabs a Vesta longsword.',
+	'{1} takes a Ghrazi Rapier from inside the Trinity Outpost.',
+	'{1} finds a Crate with an Infernal cape.',
+	'{1} takes a AGS from inside the Moser Settlement.',
+	'{1}, {2}, and {3} work together to get as many supplies as possible.',
+	'{1} runs away with a pair of Eternal boots and some sharks.',
+	'{1} snatches a vial of Super restore and a Stamina potion.',
+	'{1} finds a supply box full of Granite mauls & supplies.',
+	'{1} searches a supply box, not realizing it is empty.',
+	'{1}, {2}, {3}, and {4} share everything they gathered before running.',
+	'{1} retrieves a Staff of the Dead from the Debtor hideout .',
+	"{1} grabs a Stamina potion while {2} gets Ahrim's robes.",
+	'{1} scares {2} away from the Moser Settlement.',
+	'{1} grabs a Dragonfire shield from the ground on The Mountain.',
+	'{1} snatches a pair of Dragon claws.',
+	'{1} grabs a lone pair of pants.',
+	'{1} trips over while running to a building, {2} picks them up, they run off together.'
+].map(string => new LastManStandingUsage(string));
+
+export const LMS_FINAL = [
+	'{1} eats a shark.',
+	'{1} attacks {2}, but they manage to escape.',
+	'{1} chases {2}.',
+	'{1} runs away from {2}.',
+	'{1} defeats {2} in a fight, but spares their life.',
+	'{1} and {2} work together for now.',
+	'{1} makes a crossbow.',
+	'{1} travels to higher ground.',
+	'{1} forces {2} to eat pant.',
+	'{1} searches for a bloody chest.',
+	'{1} uses their bloodier key and recieves a full inventory of food.',
+	'{1K} forces {2D} to eat pant. {2D} chokes and dies.',
+	"{1K} throws a dragon knife into {2D}'s head.",
+	'{1D} begs for {2K} to kill them. They reluctantly oblige, killing {1D}.',
+	'{1K} and {2K} work together to spec out {3D}.',
+	'{1K} decapitates {2D} with an AGS spec.',
+	'{1K} sets {2D} on fire with Flames of Zamorak.',
+	"{1K} DDS specs {2D} while they're purchasing supplies.",
+	'{1K} severely injures {2D}, but puts them out of their misery.',
+	'{1K} severely injures {2D} and leaves them to die.',
+	"{1K} bashes {2D}'s head in with a Dragon mace.",
+	"{1K} throws a dragon knife into {2D}'s chest.",
+	'{1D} is unable to convince {2K} to not kill them.',
+	'{1K} convinces {2D} to not kill them, only to kill {2D} instead.',
+	'{1D} unknowingly runs into the fog and dies.',
+	'{1K} throws a chinchompa, killing {2D}.',
+	'{1K} throws a chinchompa, killing {2D}, and {3D}.',
+	'{1D} dies from poison.',
+	'{1D} dies from venom.',
+	'{1D} dies from disease.',
+	"{1K} kills {2D} with a Morrigan's throwing axe.",
+	'{1D} dies in the fog trying to escape the arena.',
+	'{1D} accidently detonates a chinchompa trying to throw it.',
+	'{1K} ambushes {2D} and kills them.',
+	'{1K} severely slices {2D} with an AGS spec.',
+	'{1K} shoots {2D} with a Dark bow.',
+	"{1K} stabs {2D} in the back with a Vesta's longsword.",
+	'{1K} kills {2D} with a Ghrazi rapier.',
+	'{1K} repeatedly stabs {2D} to death with Draggon dagger.'
```

Comment:
'{1D} Talks about how good they are while fighting {2K} and is promptly stacked out by {2K}',

Reply:
as an addition